src/oc/hoc_oop.cpp

```diff
@@ -605,7 +603,6 @@ void hoc_newobj(void) { /* template at pc+1 */
         hoc_pushobj(obp);
 #if USE_PYTHON
     } else { /* Assignment to OBJECTTMP not allowed */
-        Object* o = hoc_obj_look_inside_stack(narg);
```

Comment:
seems to have side effects, like detecting errors, etc.

Reply:
I put it back after checking the function. It needs to throw an error if the item is not `Object*` or `Object**`.